appveyor.yml

```diff
@@ -4,10 +4,6 @@ init:
 - ps: |
       $version = new-object System.Version $env:APPVEYOR_BUILD_VERSION
       $env:flowVersion = "{0}.{1}.{2}" -f $version.Major, $version.Minor, $version.Build
-      if ($env:APPVEYOR_REPO_BRANCH -eq "dev")
-      {
-        $env:prereleaseTag = "{0}.{1}.{2}.{3}" -f $version.Major, $version.Minor, $version.Build, $version.Revision
-      }
```

Comment:
Why is this removed? We need to add the revision version for prereleases.

Reply:
not removed, just replaced with $APPVEYOR_BUILD_VERSION, they are exactly the same

Reply:
That's fine make sure you test it, if I remember correctly when I updated this, that pipeline variable returned only the build number part of the version.

Reply:
see line 5 above, `$env:prereleaseTag` was derived from `$env:APPVEYOR_BUILD_VERSION`